index.html

```diff
@@ -299,5 +299,31 @@ <h4 class="ico">LET’S CONNECT</h4>
           </section>
         </div>
       </section>
+      <section class="contact-form">
+        <div class="contact-section">
+          <section class="contact-content">
+            <h1>Contact me</h1>
+           <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+       
+         <form action="https://formspree.io/f/mdojqrgq " method="post">
+           <ul class="form-sections">
+             <li>
+               <input  required max="30" type="text" name="user-name" placeholder="Name" class="form-input">
```

Comment:
- Hey, let's kindly improve on our max to `maxlength="30"` as this is the correct way of adding the maxlength attribute to enforce the input's validation.

Reply:
Fixed.